src/main/resources/templates/db_scripts/migratePrecedingSucceedingTitles.sql

```diff
@@ -0,0 +1,22 @@
+START TRANSACTION;
+
+INSERT INTO ${myuniversity}_${mymodule}.preceding_succeeding_title (id, jsonb,
+	   creation_date, created_by, precedingInstanceId, succeedingInstanceId)
+  SELECT id, jsonb_build_object(
+    'id', jsonb->'id',
+	'metadata', jsonb_build_object(
+		'createdDate', jsonb->'metadata'->'createdDate',
+		'updatedDate', jsonb->'metadata'->'updatedDate'),
+    'precedingInstanceId', jsonb->'superInstanceId',
+	'succeedingInstanceId', jsonb->'subInstanceId'),
+   creation_date, created_by, superInstanceId, subInstanceId
+  FROM ${myuniversity}_${mymodule}.instance_relationship as ir
+  WHERE ir.jsonb->>'instanceRelationshipTypeId'='cde80cc2-0c8b-4672-82d4-721e51dcb990';
+
+DELETE FROM ${myuniversity}_${mymodule}.instance_relationship WHERE
+  jsonb->>'instanceRelationshipTypeId'='cde80cc2-0c8b-4672-82d4-721e51dcb990';
+
+DELETE FROM ${myuniversity}_${mymodule}.instance_relationship_type WHERE
+  id='cde80cc2-0c8b-4672-82d4-721e51dcb990';
```

Comment:
I think we should postpone removing old data, because UI still uses the old way. We should fill a separate jira to remove them after whole migration.

Reply:
It seems to me it will be better to remove old data in the separate story. 
I've created https://issues.folio.org/browse/MODINVSTOR-451